src/main/java/course_project/SeaBattle.java

```diff
@@ -0,0 +1,95 @@
+package course_project;
+
+import lombok.Data;
+import java.util.Collections;
+import java.util.Scanner;
+import java.util.TreeMap;
+
+@Data
+public class SeaBattle {
```

Comment:
I couldn't play the game... it just takes so long to start, to place ships manually for 2 players, especially because I need to input all the cells of a ship... Please refactor, I'll check then

Reply:
Fixed!